### ISSUE TITLE
Heavy slowdown at the end of Pokémon GBA to DS migration

### ISSUE THREAD

**Issue by rzumer** [Arisotura/melonDS#567]
This is a minor issue left over from #550. It only lasts about a second, and as far as I could tell it had to do with many writes being queued to the GBA cartridge at once. Commenting out the actual writes to file did not seem to improve it, so my guess is that it's a timing (emulation) issue.

**Comment by RSDuck** [comment]
it might as well be the DS game writing to it's save file

**Comment by rzumer** [comment]
Yeah, that's also possible. I haven't tried intercepting those yet.

**Comment by rzumer** [comment]
#925 hasn't solved the issue (it might be a little bit faster?), so it doesn't seem like NDS cartridge writes were the main problem, or the only one.
